Source/OrchBase/Composer.cpp

```diff
@@ -812,9 +1458,23 @@ int Composer::generateSupervisor(ComposerGraphI_t* builderGraphI)
     int devIdx = 0; // Faster than using std::distance
     builderGraphI->supevisorDevIVect.clear();       // Sanity clear
     builderGraphI->devISuperIdxMap.clear();         // sanity clear
-
-    supervisor_cpp << "const std::vector<SupervisorDeviceInstance_t> ";
-    supervisor_cpp << "Supervisor::DeviceVector = { ";
+    
+    
+      
+    // Create the Supervisor's DeviceVector binary blob
+    std::stringstream supervisor_binFName;
+    supervisor_binFName << builderGraphI->outputDir << "/" << GENERATED_PATH;
+    supervisor_binFName << "/supervisor.bin";
+    std::string supervisor_binFNameStr = supervisor_binFName.str();
+
+    supervisor_bin.open(supervisor_binFNameStr.c_str());
+    if(supervisor_bin.fail()) // Check that the file opened
+    {                 // if it didn't, tell logserver and exit
+        //TODO: Barf
+        //par->Post(816, vars_hFName.str(), OSFixes::getSysErrorString(errno));
```

Comment:
Why not post here? 🍠

Reply:
This comes back to the "Composer doesn't know about post" and implementing the error string return seemed like a "jam tomorrow" thing given everything else we have on.